internal/fetch/jobDescription_test.go

```diff
@@ -33,14 +39,54 @@ func Test_parseJobDescription(t *testing.T) {
 			},
 		},
 	} {
-		t.Run(tc.jobDescription, func(t *testing.T) {
+		s.Run(tc.jobDescription, func() {
 			got, err := ParseJobDescription(tc.defaultPipelineID, tc.jobDescription)
 			if tc.wantErr {
-				require.Error(t, err, tc.jobDescription)
+				s.Require().Error(err, tc.jobDescription)
 				return
 			}
-			require.NoError(t, err, tc.jobDescription)
-			assert.Equal(t, tc.want, got, tc.jobDescription)
+			s.Require().NoError(err, tc.jobDescription)
+			s.Assert().Equal(tc.want, got, tc.jobDescription)
+		})
+	}
+}
+
+func (s *JobDescriptionSuite) TestJobDescription_External() {
+	for _, tc := range []struct {
+		name           string
+		jobDescription JobDescription
+	}{
+		{
+			name: "sd@123:fooBar",
+			jobDescription: JobDescription{
+				PipelineID: 123,
+				JobName:    "fooBar",
+			},
+		},
+	} {
+		s.Run(tc.name, func() {
+			got := tc.jobDescription.External()
+			s.Assert().Equal(tc.name, got)
```

Comment:
(nit) the recommended order is `got, then want` - both in comparisons as well as asserts and error messages

Reply:
I'm sorry to say, that's wrong - it's expected, actual, msgAndArgs...

https://godoc.org/github.com/stretchr/testify/assert#Assertions.Equal

Reply:
ok - never mind then.  Too bad the `assert` library deviates from the Go conventions as per https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures as well as the example in https://golang.org/pkg/testing/ - `t.Errorf("Abs(-1) = %d; want 1", got)`...